### ISSUE TITLE
Test webgl visualisations on raspberry pi ESTIMATE 3h

### ISSUE THREAD

**Issue by rpkarlsson** [Visualisering/Visualisering-frontend#28]
Testing is done on a Raspberry Pi 2 Model B

**Comment by sk222sw** [comment]
Visualisations didn't work in the chromium browser on a rpi last week, but changes have been made and this is ready to be tested again, am I right? @mw222rs, @rk222ev. Just checking before I crawl under my desktop to fetch the cables from my computer and plug them into my rpi.

**Comment by rpkarlsson** [comment]
@mw222rs changed Object.assign to use Lodash. If I remember correctly this was something that broke everything. The problem we had should be fixed.

**Comment by fyrkant** [comment]
Yeah it should, hopefully work now. I have a rpi 3 on the way to me from England so I can test on that later in the week!
